<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComboStatusPanel</class>
 <widget class="QDockWidget" name="ComboStatusPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-Status</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Switch 1 Status:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="transmissionBox">
      <property name="text">
       <string>Transmission</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="reflectionBox">
      <property name="text">
       <string>Reflection</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Switch 2 Status:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="networkAnalyzerBox">
      <property name="text">
       <string>Network Analyzer</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="digitizerBox">
      <property name="text">
       <string>Digitizer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Cavity Length ( in )</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLCDNumber" name="cavLengthLCD"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Iteration Number</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLCDNumber" name="iterNumberLCD"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>LO Frequency (MHz)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLCDNumber" name="LOLCD"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
